R/pmcmc.R

```diff
@@ -109,8 +109,22 @@ pmcmc <- function(data,
                   steps_per_day = 4) {
   
   # test pars_init input
-  correct_format <- function(pars) {
-    is.list(pars) & setequal(names(pars), c('beta', 'start_date'))
+  correct_format <- function(pars, input = "") {
```

Comment:
I would suggest splitting this in two, as this new form doesn't share any of the code. You could leave the simple format of `correct_format()` you had previously as 'correct_format_vec()`, and add `correct_format_mat()` for the new check.